src/EditorFeatures/VisualBasicTest/CodeActions/ConvertConversionOperators/ConvertConversionOperatorFromCTypeTests.vb

```diff
@@ -0,0 +1,116 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports Microsoft.CodeAnalysis.CodeRefactorings
+Imports Microsoft.CodeAnalysis.VisualBasic.CodeRefactorings.ConvertConversionOperators
+
+Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.CodeRefactorings.ConvertConversionOperators
```

Comment:
do we not have the equivalent of VerifyVb here?

Reply:
I looked into the `ConvertIfToSwitchTests`:
https://github.com/dotnet/roslyn/blob/a34f425d15b9acd35e749d99b31c682fbb8ddf4b/src/EditorFeatures/CSharpTest/CodeActions/ConvertIfToSwitch/ConvertIfToSwitchTests.cs#L49-L54

The VB tests there don't use some kind of *VerifyVB*, so I assume there is no such thing.
https://github.com/dotnet/roslyn/blob/a34f425d15b9acd35e749d99b31c682fbb8ddf4b/src/EditorFeatures/VisualBasicTest/CodeActions/ConvertIfToSwitch/ConvertIfToSwitchTests.vb#L17-L19

Reply:
not all tests have been converted over to the new SDK.  but we want new tests to use the new SDK approach :)